Client/Coordinators/TabTray/RemoteTabsCoordinator.swift

```diff
@@ -53,4 +55,17 @@ class RemoteTabsCoordinator: BaseCoordinator, RemoteTabsPanelDelegate {
     func didFinish(from childCoordinator: Coordinator) {
         remove(child: childCoordinator)
     }
+
+    // MARK: - QRCodeNavigationHandler
+    func showQRCode(delegate: QRCodeViewControllerDelegate, rootNavigationController: UINavigationController?) {
```

Comment:
@FilippoZazzeroni moving to RemoteTabsCoordinator fix the issue of early dismissal, thanks once again

Reply:
@yoanarios perfect you re welcome. Nice to hear that.